e2e-tests/cypress/integration/mvp-eligibility-check-decision-date.feature

```diff
@@ -0,0 +1,32 @@
+@wip
+Feature: Check that decision date is eligible to commence an appeal
+  Note - This feature is intended to support redirection to ACP Service as part of initial MVP
+
+  Scenario: Eligibility decision dates less than 12 weeks old can proceed
+    Given the application decision date is requested
+    When a decision date is provided that is less than 12 weeks old
+    Then the appeal can be commenced
+
+  Scenario: Eligibility decision dates older than 12 weeks cannot proceed
+    Given the application decision date is requested
+    When a decision date is provided that is more than 12 weeks old
+    Then the decision date is highlighted as beyond the deadline for appeal
+
```

Comment:
I'm fine using highlighted in both scenario but idk if it was what you meant or not. When the decision date expired you are redirected to a specific page. This is not an error message like it would be if the date is invalid.

Reply:
I am not sure what you mean by 'both' here. This highlighted scenario is a key behaviour and I don't expect it to be handled as an error. Instead it will provide an end of journey page similar to this.
<img width="762" alt="Screenshot 2021-01-07 at 08 35 52" src="https://user-images.githubusercontent.com/56199648/103870268-5c297e00-50c3-11eb-98a6-802474079cca.png">